index.html

```diff
@@ -54,9 +54,85 @@ <h5 >I help bring awesome web experiences to life, if you have a project in mind
         </main>
 
      <!-- Recent Works -->
-     <section></section>
+        <section class="my_work" id="Portfolio">
+            <h3 class="works_tittle">My recent Works</h3>
+            <div class="grid-container">
+                <div class="work_top">
+                    <div class="main_card">
+                    <h3 id="featured">Multi-Post Stories</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul id="tags_featured"><li>css</li><li>html</li><li>bootstrap</li><li>ruby</li></ul>
+                    <a href="#" class="project-button">See Project</a>
+                    </div>
+                    <img src="images/Img_Placeholder.png" alt="">
+                </div>
+                <div class="work-1">
+                    <div class="card_content">
+                        <h3>Website Portfolio</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore </p>
+                        <ul id="tags"><li>css</li><li>html</li><li>bootstrap</li><li>ruby</li></ul>
+                        
+                    </div>
+                </div>
+                <div class="work-2">
+                    <div class="card_content">
+                        <h3>Data Dashboard</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore</p>
+                        <ul id="tags"><li>css</li><li>html</li><li>bootstrap</li><li>ruby</li></ul>
+                        
+                    </div>
+                </div>
+                <div class="work-3">
+                    <div class="card_content">
+                        <h3>Brand website</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore</p>
+                        <ul id="tags"><li>css</li><li>html</li><li>bootstrap</li><li>ruby</li></ul>
+                        
+                    </div>
+                </div>
+                <div class="work-4">
+                    <div class="card_content">
+                        <h3>Shopping Website</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore</p>
```

Comment:
- [ ] You did great implementing the changes required by the previous reviewer however, the boutons to show the projects are not displayed. Kindly fix that issue.
![review](https://user-images.githubusercontent.com/53347966/219899547-e1b916e0-2b18-4fb6-8f95-b5428a10d125.JPG)

Reply:
@algerina Thank you, buttons have been added